src/components/Toolbar/VisualizationTypeSelector/VisualizationTypeSelector.js

```diff
@@ -89,7 +95,7 @@ const VisualizationTypeSelector = ({ visualizationType }) => {
 }
 
 VisualizationTypeSelector.propTypes = {
-    visualizationType: PropTypes.oneOf(Object.keys(visTypeMap)),
+    visualizationType: PropTypes.oneOf(Object.keys(visTypeDescriptions)),
```

Comment:
Even though all `visTypes` would have be listed in `visTypeDescriptions`, the source of truth here should be `visTypes`, right? So e.g. 
```suggestion
    visualizationType: PropTypes.oneOf(visTypes.map(visType => visType.type)),
```

Reply:
Changed.